### PR TITLE
made css

### PR DESCRIPTION
**Pull request by JasonLin43212**

### CODE REVIEW

File: dist/dev/css/main-index.css [hunk 1]
```diff
@@ -17,52 +17,35 @@ body {
...
 
-    .col-lg-3 {
-        width: 256px;
-    }
+}
 
-    .col-lg-4 {
-        width: 346px;
+@media (min-width: 992px) and (max-width: 1159px){
+    .col-md-1, .col-md-2, .col-md-3, .col-md-4, .col-md-5, .col-md-6, .col-md-7,
```
Comment (jkao1) [inline dist/dev/css/main-index.css]: You have a hole between 1159px and 1200px.

Comment (JasonLin43212) [reply dist/dev/css/main-index.css]: but if i made the max width 1200px, then how would it know which width to use?

Comment (JasonLin43212) [reply dist/dev/css/main-index.css]: also this https://stackoverflow.com/questions/13241531/what-are-the-rules-for-css-media-query-overlap

Comment (jkao1) [reply dist/dev/css/main-index.css]: First of all that doesn't justify the max-width 1159px. The min-width for lg is 1200, but you set the max-width of md to 1159?

Comment (JasonLin43212) [reply dist/dev/css/main-index.css]: oh shit you are right

Comment (jkao1) [reply dist/dev/css/main-index.css]: lol; change and i'll merge

File: dist/dev/css/main-index.css [hunk 2]
```diff
@@ -17,52 +17,35 @@ body {
...
     }
 
-    .col-lg-3 {
-        width: 256px;
-    }
+}
 
-    .col-lg-4 {
-        width: 346px;
+@media (min-width: 992px) and (max-width: 1200px){
```
Comment (jkao1) [inline dist/dev/css/main-index.css]: From your previous logic, shouldn't max-width here be 1199px?

File: dist/dev/css/main-index.css [hunk 3]
```diff
@@ -17,52 +17,35 @@ body {
...
     }
 
-    .col-lg-6 {
-        width: 526px;
-    }
+}
 
-    .col-lg-7 {
-        width: 616px;
+@media (min-width: 768px) and (max-width: 992px) {
```
Comment (jkao1) [inline dist/dev/css/main-index.css]: Same here, to 991px?

File: dist/dev/css/main-index.css [hunk 4]
```diff
@@ -17,52 +17,35 @@ body {
...
     }
 
-    .col-lg-9 {
-        width: 796px;
-    }
+}
 
-    .col-lg-10 {
-        width: 886px;
+@media (max-width: 768px) {
```
Comment (jkao1) [inline dist/dev/css/main-index.css]: And here?

File: dist/dev/css/main-index.css (hunk 2 above)
Comment (JasonLin43212) [reply dist/dev/css/main-index.css]: i changed it because i thought u wanted to remove the hole between 1159 and  1200

Comment (jkao1) [reply dist/dev/css/main-index.css]: If media queries override, there isn't a point to setting max-width. Just stack the media queries in a certain order such that it'll override correctly. 
```
/* Extra Small Devices, Phones */ 
@media only screen and (min-width : 480px) {

}

/* Small Devices, Tablets */
@media only screen and (min-width : 768px) {

}

/* Medium Devices, Desktops */
@media only screen and (min-width : 992px) {

}

/* Large Devices, Wide Screens */
@media only screen and (min-width : 1200px) {

}
```

Comment (JasonLin43212) [reply dist/dev/css/main-index.css]: do i need the "only screen" part as well?

Comment (jkao1) [reply dist/dev/css/main-index.css]: Just put it. We can get some nerd cred

Comment (JasonLin43212) [reply dist/dev/css/main-index.css]: wait, i thought that the small screens had a max width of 768? or are we changing it to a min width of 480?

Comment (jkao1) [reply dist/dev/css/main-index.css]: leave it at max-width

**Pull request merged**